app/services/clinic_finder/locator.rb

```diff
@@ -85,27 +71,9 @@ def filter_by_params(clinics, gestational_age, naf_only, medicaid_only)
 
     def set_geocoder
```

Comment:
What exactly is being returned by this function if `GOOGLE_GEO_API_KEY` isn't set? Just an empty object?

Reply:
It returns a `Geokit::Geocoders::GoogleGeocoder` without an API key set. The class is nice and malleable and will let you init it without a key, and (I think) google lets you have like one or two free calls per IP before it's like 'seriously, get a token pls'. But it's esp nice for us because it'll respond to everything we expect it to regardless of whether there's a key set.